Source/Tests/UnitTestGpuApi/App/UnitTestApp.cpp

```diff
@@ -13,6 +13,40 @@ namespace UNITTEST_GPUAPI
 		: App(InClientSize, CommandLine)
 	{
 		Editor = MakeUnique<UnitTestEditor>(AppClientSize);
+
```

Comment:
How to choose an api backend should be placed in a more generic location, rather than rewriting every program. There also doesn't seem to be a default api backend according to a specific platform.